### ISSUE TITLE
Build failed: unsupported GOOS/GOARCH pair darwin/386

### ISSUE THREAD

**Issue by seafarer** [Automattic/go-search-replace#25]
```Bash
1 errors occurred:
--> darwin/386 error: exit status 2
Stderr: cmd/go: unsupported GOOS/GOARCH pair darwin/386

make: *** [build] Error 1
```

It looks like this might be the culprit: https://go-review.googlesource.com/c/tools/+/227552/

>The darwin/386 port has been removed per golang.org/issue/37610.
TestSizes needs an operating system that has both amd64 and 386 ports,
so darwin no longer qualifies. NetBSD has had its 386 port restored
recently in golang/go#31726, so it can be used instead.

I got myself up and running by deleting everything in the makefile except my own os, but dont know Go well enough to submit a PR.

**Comment by joshbetz** [comment]
@seafarer Does this latest commit fix it?

**Comment by seafarer** [comment]
@joshbetz unfortunately no. Same error. I do have gox installed.
```Bash
$ which gox
$ /Users/<user>/go/bin/gox
```

**Comment by pjhoberman** [comment]
I have the same issue as @seafarer, and the same solution (removing `-os="darwin" -os="windows"` from the Makefile) worked for me.

**Comment by youngjuning** [comment]
@pjhoberman just remove -os="darwin"

**Comment by jblz** [comment]
Fixed in #28
